packages/template-retail-react-app/app/utils/site-utils.js

```diff
@@ -18,7 +18,7 @@ export const resolveSiteFromUrl = (url) => {
     if (!url) {
         throw new Error('URL is required to find a site object.')
     }
-    const {pathname, search} = new URL(absoluteUrl(url))
+    const {pathname, search} = getPathnameAndSearch(url)
```

Comment:
small refactor to not use new URL since it needs `origin` to construct the url object for `resolveSiteFromUrl` and `getParamsFromPath` funcs. We are not using the origin/host for these funcs, we only are interested in the pathname and seach params. 
This is not a breaking change because there is not change in signature or current behaviors